static/app/views/dashboardsV2/widgetCard/widgetQueries.tsx

```diff
@@ -531,7 +475,7 @@ class WidgetQueries extends Component<Props, State> {
     this.setState({loading: true, errorMessage: undefined, queryFetchID});
 
     if (['table', 'world_map', 'big_number'].includes(widget.displayType)) {
-      this.fetchEventData(queryFetchID);
+      this.fetchTableData(queryFetchID);
```

Comment:
I renamed this because it was easy and moves us closer to thinking of "fetchTimeseriesData" and "fetchTableData" as we extract similar patterns. I can revert this or change the name if we think this is a change for another PR.